code/modules/projectiles/guns/energy/pulse.dm

```diff
@@ -24,11 +24,11 @@
 /obj/item/weapon/gun/energy/pulse_rifle/destroyer
 	name = "pulse destroyer"
 	desc = "A heavy-duty, pulse-based energy weapon. Because of its complexity and cost, it is rarely seen in use except by specialists."
-	cell_type = /obj/item/weapon/cell/super
-	fire_delay = 25
+//	cell_type = /obj/item/weapon/cell/super
+	fire_delay = null		//Aspawn only gun doesn't really need to be balanced against the station/antags

```

Comment:
I don't think this change should be made. Yes, it's an adminspawn gun, but just because it's adminspawn doesn't mean it should be brokenly overpowered when it wasn't before.

The SAW is powerful, but it's not obscene. It doesn't fire 60 damage bullets at a billion rounds a second because loladminboose. It's just a powerful gun that is used in some events, more powerful than what is normally available.

If I want to admin abuse with the intention of fucking shit up I'll spawn a meteor gun.